lib/go-tc/server_capability.go

```diff
@@ -0,0 +1,9 @@
+package tc
+
+// ServerServerCapabilityNullable
+type ServerServerCapabilityNullable struct {
```

Comment:
The "nullable" concept is a legacy thing from the TO Go client using non-nullable structs and the TO API backend using nullable structs. Since this is a new endpoint with new client funcs we can get rid of the "Nullable" suffix since both sides are using the same struct.